### PR TITLE
Mon 45 new device tree model from api

### PR DESCRIPTION
**Pull request by VVojtekMecki**
- Installed axios
- fixed api endpoint for data tree model
- connects to api and fetches json
- parse json do device model
- integration with charts

### CODE REVIEW

File: src/main/webclient/src/types/deviceModel.ts [hunk 1]
```diff
@@ -87,6 +91,112 @@ interface IDeviceModel {
...
+        bridges.push(
+          new Bridge(
+            topLevelDevice.id,
+            topLevelDevice.status,
+            new Date(topLevelDevice.lastPinged),
+            devices[1].devices
+              .filter(
+                (device) =>
+                  device.deviceType === deviceType.gateway &&
+                  topLevelDevice.children?.includes(device.id)
```
Comment (sasankasa) [inline src/main/webclient/src/types/deviceModel.ts:128]: fix ode mnie!

File: src/main/webclient/src/api/api-client.ts [hunk 2]
```diff
@@ -23,4 +28,17 @@ export class APIClient {
       ]),
     ]);
   };
+  public static getUpdatedDeviceModel = () => {
+    return axios
+      .get(apuURL)
+      .then((response) => {
+        const data: DeviceTreeModelJson = response.data;
```
Comment (sasankasa) [inline src/main/webclient/src/api/api-client.ts:35]: pierwszy hot fix ode mnie :)

**Review by sasankasa — APPROVED** [review]
Good job, O i W :)

File: src/main/webclient/src/types/deviceModel.ts (hunk 1 above)
Comment (VVojtekMecki) [reply src/main/webclient/src/types/deviceModel.ts:128]: Tak

File: src/main/webclient/src/api/api-client.ts (hunk 2 above)
Comment (VVojtekMecki) [reply src/main/webclient/src/api/api-client.ts:35]: Tak

**Pull request merged**